src/components/Routes/PatientsEdit.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
```

Comment:
Patients Edit and Create new will be the same page. Check how this is implemented in the Visits Pages